src/history/historyReducer.ts

```diff
@@ -1,16 +1,21 @@
+import { Reducer } from 'redux'
+
 import {
-  HISTORY_RECEIVED_ACTION,
   CHANGE_DATE_FILTER_ACTION,
-  CHANGE_PRICE_FILTER_ACTION
+  CHANGE_PRICE_FILTER_ACTION,
+  HISTORY_RECEIVED_ACTION,
+  HistoryActions
 } from './historyActions'
+import { HistoryState } from './types'
 
-const initialState = {
+const initialState: HistoryState = {
+  company: 'aapl',
   history: [],
   dateFilter: 'ytd',
   priceFilter: 'close'
 }
 
-export default (state = initialState, action) => {
+const history: Reducer<HistoryState, HistoryActions> = (state = initialState, action) => {
```

Comment:
Maybe rename this to historyReducer? history appears both as the function and as the object attribute in line 21.